region/crateria/central.json

```diff
@@ -1082,7 +1109,9 @@
           },
           "locks": [
             {
-              "unlock": [ "ZebesAwake", "Kill All Enemies" ]
+              "lockType": "killEnemies",
+              "lock": ["ZebesAwake"],
```

Comment:
This door is locked prior to activating ZebesAwake as well

Reply:
It sure is. What I get for not checking.